src/regions/KeyPointRegion.js

```diff
@@ -105,10 +105,22 @@ const Model = types
       };
     },
     get canvasX() {
-      return isFF(FF_DEV_3793) ? self.parent?.internalToCanvasX(self.x) : self.x;
+      const isSnap = self.tag.snap === 'pixel';
```

Comment:
It won't work with `KeyPoint` as separated tool as it may have no tag which is getter for a related label. So I think it might be something like `self.control.snap`